content/200-concepts/100-components/02-prisma-client/053-middleware/100-soft-delete-middleware.mdx

```diff
@@ -598,3 +604,15 @@ prisma.post.findMany({ where: { includeDeleted: true } })
 > **Note**: You would still need to write middleware.
 
 We **✘ do not** recommend this approach as it pollutes the schema with fields that do not represent real data.
+
+## Limitations
+
+The soft delete sample on this page is not a full implementation. In the options above, we explain some of the limitations. Other limitations include the following:
+
+- `findMany` with options such as `some` and `none` does not work
+- Most nested queries, such as `findMany({ where: {}, include: { }` and `update({data: { posts: { deleteMany: { } } } })` do not work
```

Comment:
Missing brackets... maybe this? (please check I haven't got it wrong!)

```suggestion
- Most nested queries, such as `findMany({ where: {}, include: {}})` and `update({data: { posts: { deleteMany: { } } } })` do not work
```

Reply:
Yep, you're right!